docs/source/tutorials/token_classification/tutorial-p2.ipynb

```diff
@@ -0,0 +1,522 @@
+   "source": [
+    "## 1. Install the required dependencies "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "a1349304",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "from termcolor import colored \n",
+    "from cleanlab.token_classification.filter import find_label_issues \n",
+    "from cleanlab.token_classification.rank import get_label_quality_scores\n",
+    "from utils import * \n",
+    "# change to this after token_classification.rank is pushed to the official package \n",
+    "# from cleanlab.token_classification.rank import get_label_quality_scores "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9ad75b45",
+   "metadata": {},
+   "source": [
+    "## 2. Get `pred_probs` and `labels` \n",
+    "\n",
+    "`pred_probs` are out-of-sample model-predicted probabilities of the CoNLL-2003 dataset (including training, development, and testing dataset), obtained via cross-validation. To detect potential labels issues, we first get `pred_probs` and `labels`, which are both in nested-list format, such that: \n",
+    "\n",
+    "- `pred_probs` is a list of `np.arrays`, such that `pred_probs[i]` is the model-predicted probabilities for the tokens in the i'th sentence, and has shape `(N_i, K)`, where `N_i` is the number of word-level tokens of the `i`'th sentence. Each row of the matrix corresponds to a token `t` and contains the model-predicted probabilities that `t` belongs to each possible class, for each of the K classes. The columns must be ordered such that the probabilities correspond to class 0, 1, ..., K-1. \n",
+    "        \n",
+    "- `labels` is a list of lists, such that `labels[i]` is a list of token labels of the `i`'th sentence. Same format requirements as `cleanlab.rank.get_label_quality_scores`. \n",
+    "\n",
+    "Here, indicies are a tuple `(i, j)` unless otherwise specified, which refers to the `j`'th word-level token of the `i`'th sentence. Given that each sentence has different number of tokens, we store `pred_probs` and `labels` as `.npz` files, which can be easily converted to dictionaries. Use `read_npz` to retrieve `pred_probs` and `labels` in nested-list format. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "519cb80c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "labels = read_npz('labels.npz') \n",
+    "pred_probs = read_npz('pred_probs.npz') "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1dc3150f",
+   "metadata": {},
+   "source": [
+    "## 3. Use cleanlab to find potential label issues \n",
+    "\n",
+    "Based on the given labels and out-of-sample predicted probabilities, cleanlab can quickly help us identify label issues in our dataset. Here we request that the indices of the identified label issues be sorted by cleanlab’s self-confidence score, which measures the quality of each given label via the probability assigned to it in our model’s prediction."
```

Comment:
You should explain what each tuple (and index of the tuple) in `issues` means

Reply:
There's a few sentences right above that (in section 2) that explains the tuples. I place it there to justify `.npz` storage, but do you want me to move to section 3?